app/assets/javascripts/organisation-autocomplete.js

```diff
@@ -5,16 +5,118 @@ window.GOVUKAdmin.Modules = window.GOVUKAdmin.Modules || {};
 
   Modules.OrganisationAutocomplete = function () {
     this.start = function ($element) {
-      accessibleAutocomplete.enhanceSelectElement({
-        selectElement: $element.find('#organisations')[0],
-        showAllValues: true,
-        autoselect: false,
-        dropdownArrow: dropdownArrow
-      });
-    };
 
-    function dropdownArrow() {
-      return '<span class="caret"></span>';
-    }
+      var autocompleteTemplateHtml = createAutocompleteTemplate();
+
+      $element
```

Comment:
If you're not moving this into the HTML, could we please break up this slightly monolithic nested jQuery chain? It would be nice to move it into a series of single-responsibility functions that describe what they're doing, eg `addAddOrganisationButton` (definitely want a better name than that); `addSelectElementsForSelectedOptions`

Reply:
👍